vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction_test.go

```diff
@@ -60,11 +69,13 @@ func TestEvictReplicatedByController(t *testing.T) {
 	testCases := []struct {
 		replicas           int32
 		evictionTollerance float64
+		vpa                *vpa_types.VerticalPodAutoscaler
 		pods               []podWithExpectations
 	}{
 		{
 			replicas:           3,
 			evictionTollerance: 0.5,
+			vpa:                generateVpa(),
```

Comment:
I'd just put `test.VerticalPodAutoscaler().WithContainer("any").Get()` here. I think we need `generatePod` because we want pods to have different names but we're not doing that with VPAs here

Reply:
Done.